bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -449,4 +449,86 @@
 		<category>thermostat</category>
 	</channel-type>
 
+	<channel-type id="smokeDetection">
+		<item-type>Switch</item-type>
+		<label>Smoke Detected</label>
+		<description>Smoke detection sensor state.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="sensorFault">
+		<item-type>Switch</item-type>
+		<label>Sensor Fault Mode</label>
+		<description>Sensor activated fault mode.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="maintenanceStatus">
+		<item-type>Switch</item-type>
+		<label>Maintenance neccessary</label>
```

Comment:
Can you make sure that every word from this label starts with a capital this is the default in ESH / openHAB.

See for the exact rules: https://capitalizemytitle.com/

Reply:
Fixed 👍